misc/angular-bootstrap-prettify.js

```diff
@@ -0,0 +1,323 @@
+'use strict';
```

Comment:
I had to add this file directly to our code base. See line 209: https://github.com/angular-ui/bootstrap/issues/4315